doc/DeveloperGuide.md

```diff
@@ -88,6 +88,31 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename person tag
+
+**MSS**
+
+1. User requests to list persons
+2. AddressBook shows a list of persons
+3. User requests to rename an existing tag of a person with a new tag
+4. AddressBook update the person with a new tag
+Use case ends.
+
+**Extensions**
+
+2a. The list is empty
+
+> Use case ends
+
+3a. The given index is invalid
+
+> 3a1. AddressBook shows an error message <br>
+  Use case resumes at step 2
+
+4a. Person has no tag
+
```

Comment:
Perhaps this should be at step 3 too, when the user tries to rename a tag when the user has no takes at all to rename.